posts/app.py

```diff
@@ -68,13 +68,29 @@ def get_all_posts():
 
 @app.route('/v1/<post_id>', methods=['GET'])
 def get_post_by_id(post_id):
-    """Get the post with the specified ID."""
-    # serialize otherwise nonserializable ObjectIDs
+    """Get the post with the specified ID.
+    """
     post_list = find_posts_in_db(
         app.config["COLLECTION"], post_id=ObjectId(post_id))
     return serialize_posts_to_json(post_list)
 
 
+@app.route('/v1/<post_id>/delete', methods=['DELETE'])
```

Comment:
FWIW, you don't need to specify a different URL for deletion; the HTTP verb will indicate that (and it would be more aligned with REST to support DELETE at the same URL).

Reply:
Done https://github.com/knative-portability/large-events/pull/52/commits/7497b9b1b9ebe574403ac3a9f5ac0d0eb74aad27